src/index.html

```diff
@@ -17,6 +17,50 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <img src="./images/logo.png"
+           class="img"
+           alt="moyologo">
+
+      <nav class="nav">
+        <ul class="nav_list">
+          <div src="./images/Rectangle 2.svg"
+               class="rect"
+               alt="line">>
```

Comment:
The `<div>` element should not have a `src` attribute. If you intend to display an image, use an `<img>` tag instead.